doc/api/cli.md

```diff
@@ -340,6 +358,16 @@ misformatted, but any errors are otherwise ignored.
 Note that neither the well known nor extra certificates are used when the `ca`
 options property is explicitly specified for a TLS or HTTPS client or server.
 
+### `SSL_CERT_DIR=dir`
+
+If `--use-openssl-ca` is enabled, this overrides and sets OpenSSL's directory
+containing trusted certificates.
+
+### `SSL_CERT_FILE=file`
```

Comment:
I don't understand these concerns at all. These env. variables already apply for non-node programs and will continue to apply. This change does nothing for that. And warning "unauthorized code can affect child processes"??